src/XIVLauncher.Common.Unix/Compatibility/Dxvk.cs

```diff
@@ -49,7 +47,34 @@ public enum DxvkHudType
         [SettingsDescription("FPS", "Only show FPS")]
         Fps,
 
+        [SettingsDescription("DXVK Hud Custom", "Use a custom DXVK_HUD string")]
+        Custom,
+
         [SettingsDescription("Full", "Show everything")]
         Full,
+
+        [SettingsDescription("MangoHud Default", "Uses no config file.")]
+        MangoHud,
+
+        [SettingsDescription("MangoHud Custom", "Specify a custom config file")]
+        MangoHudCustom,
+
+        [SettingsDescription("MangoHud Full", "Show (almost) everything")]
+        MangoHudFull,
+    }
+
+    public enum DxvkVersion
+    {
+        [SettingsDescription("1.10.1 (default)", "The default version of DXVK used with XIVLauncher.Core.")]
+        v1_10_1,
+
+        [SettingsDescription("1.10.2", "Newer version of 1.10 branch of DXVK. Probably works.")]
+        v1_10_2,
+
+        [SettingsDescription("1.10.3", "Newer version of 1.10 branch of DXVK. Probably works.")]
```

Comment:
1.10.3 for sure works with Dalamud + GShade; it's really only 2.0 that is creating trouble